keras_cv/layers/regularization/drop_path_test.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+from keras_cv.layers import DropPath
+
+class DropPathTest(tf.test.TestCase):
```

Comment:
I'm just wondering if we should also include a more robust statistical test like the one shown [here](https://gist.github.com/sayakpaul/0abaf4aec53a73e53cf68a67f7f5f511) for layers like this. The last test in the gist is a bit incomplete but hopefully gives the idea. 

The idea is to get the results of the layer and repeat it through a number of trials and then verify if the target probability is being met or is closer (0.499 as opposed to being 0.5 for example). 

@LukeWood @AdityaKane2001

Reply:
@sayakpaul Could you please take a look at this [comment](https://github.com/keras-team/keras-cv/pull/480#discussion_r894985082) below?